packages/react-reconciler/src/ReactFiberHooks.js

```diff
@@ -1117,6 +1128,96 @@ function updateMemo<T>(
   return nextValue;
 }
 
+function mountDeferredValue<T>(
+  value: T,
+  config: TimeoutConfig | void | null,
+): T {
+  const [prevValue, setValue] = mountState(value);
+  mountEffect(
+    () => {
+      Scheduler.unstable_next(() => {
+        const previousConfig = ReactCurrentBatchConfig.suspense;
+        ReactCurrentBatchConfig.suspense = config === undefined ? null : config;
+        try {
+          setValue(value);
+        } finally {
+          ReactCurrentBatchConfig.suspense = previousConfig;
+        }
+      });
```

Comment:
Why do we not pass the `TimeoutConfig` to `Scheduler.unstable_next` in the mount case, when we do in the update case?

Reply:
Nice catch. `unstable_next` doesn't accept a second argument. It must have been a copypasta from withSuspenseConfig.

Reply:
Yeah, it was accidental copy/paste. Deleted it! :)